### ISSUE TITLE
Build fails with Cray Compiler

### ISSUE THREAD

**Issue by bavier** [trilinos/ForTrilinos#205]
Building ForTrilinos with the Cray Compiler currently fails due to the use of the `-ffree-line-length-none` compiler option in `CMakeLists.txt`, which the Cray Compiler does not recognize/support.  The best alternative option would be `-N1023` which sets the upper limit on line size to 1023 characters.  This appears to be long enough.

**Comment by aprokop** [comment]
@bavier Thanks for letting us know. I think that we may actually no longer need that option as we are able to compile with Flang that requires 132 character limit. Can you please try to compile with that option removed?

**Comment by bavier** [comment]
Yes, is works with the option completely removed.  The Cray Compiler's default line length for free form source is 255.
